master/internal/api_trials.go

```diff
@@ -682,60 +698,32 @@ func (a *apiServer) MultiTrialSample(trialID int32, metricNames []string,
 	if endBatches == 0 {
 		endBatches = math.MaxInt32
 	}
+	if maxDatapoints == 0 {
+		maxDatapoints = 200
+	}
 
-	if len(metricNames) > 0 {
-		if (metricType == apiv1.MetricType_METRIC_TYPE_TRAINING) ||
-			(metricType == apiv1.MetricType_METRIC_TYPE_UNSPECIFIED) {
-			var metric apiv1.DownsampledMetrics
-			metricMeasurements, err = trials.MetricsTimeSeries(
-				trialID, startTime, metricNames, startBatches, endBatches,
-				xAxisLabelMetrics,
-				maxDatapoints, batches, timeSeriesFilter, "training")
-			if err != nil {
-				return nil, errors.Wrapf(err, "error fetching time series of training metrics")
-			}
-			metric.Type = apiv1.MetricType_METRIC_TYPE_TRAINING
-			if len(metricMeasurements) > 0 {
-				if err = a.formatMetrics(&metric, metricMeasurements); err != nil {
-					return nil, err
-				}
-				metrics = append(metrics, &metric)
-			}
-		}
-
-		if (metricType == apiv1.MetricType_METRIC_TYPE_VALIDATION) ||
-			(metricType == apiv1.MetricType_METRIC_TYPE_UNSPECIFIED) {
-			var metric apiv1.DownsampledMetrics
-			metricMeasurements, err = trials.MetricsTimeSeries(
-				trialID, startTime, metricNames, startBatches, endBatches,
-				xAxisLabelMetrics, maxDatapoints, batches, timeSeriesFilter, "validation")
-			if err != nil {
-				return nil, errors.Wrapf(err, "error fetching time series of validation metrics")
-			}
-			metric.Type = apiv1.MetricType_METRIC_TYPE_VALIDATION
+	var timeSeriesColumn *string
```

Comment:
there are some behavior changes here were these not made to be more compatible? (mentioned in the pr description)